doc/guides/maintaining-V8.md

```diff
@@ -0,0 +1,281 @@
+* Once the fix has been merged upstream, it can be picked up during an update of the V8 branch, (see below).
+
+## Backporting to Abandoned Branches
+
+Abandoned V8 branches are supported in the Node.js V8 repository. The fix needs to be cherry-picked in the Node.js repository and V8-CI must test the change.
+
+* For each abandoned V8 branch corresponding to an LTS branch that is affected by the bug:
+    * Open a cherry-pick PR on nodejs/node targeting the appropriate *vY.x-staging* branch (e.g. *v6.x-staging* to fix an issue in V8-5.1).
+    * Increase the patch level version in v8-version.h. This will not cause any problems with versioning because V8 will not publish other patches for this branch, so Node.js can effectively bump the patch version.
+    * In some cases the patch may require extra effort to merge in case V8 has changed substantially. For important issues we may be able to lean on the V8 team to get help with reimplementing the patch.
+    * Run Node's [V8-CI](https://ci.nodejs.org/job/node-test-commit-v8-linux/) in addition to the [Node CI](https://ci.nodejs.org/job/node-test-pull-request/).
+
+An example for workflow how to cherry-pick consider the following bug: https://crbug.com/v8/5199. From the bug we can see that it was merged by V8 into 5.2 and 5.3, and not in V8 5.1 (since it was already abandoned). Since Node.js `v6.x` uses V8 5.1, the fix needed to cherry-picked. To cherry-pick, here's an example workflow:
+
+* Download and apply the commit linked-to in the issue (in this case a51f429). `curl -L https://github.com/v8/v8/commit/a51f429.patch | git apply --directory=deps/v8`. If the branches have diverged significantly, this may not apply cleanly. It may help to try to cherry-pick the merge to the oldest branch that was done upstream in V8. In this example, this would be the patch from the merge to 5.2. The hope is that this would be closer to the V8 5.1, and has a better chance of applying cleanly. If you're stuck, feel free to ping @ofrobots for help.
+* Modify the commit message to match the format we use for V8 backports. You may want to add extra description if necessary to indicate the impact of the fix on Node. In this case the original issue was descriptive enough. Example:
+```
+deps: cherry-pick a51f429 from V8 upstream
+
+Original commit message:
+  [regexp] Fix case-insensitive matching for one-byte subjects.
+
+  The bug occurs because we do not canonicalize character class ranges
+  before adding case equivalents. While adding case equivalents, we abort
+  early for one-byte subject strings, assuming that the ranges are sorted.
+  Which they are not.
+
+  R=marja@chromium.org
+  BUG=v8:5199
+
+  Review-Url: https://codereview.chromium.org/2159683002
+  Cr-Commit-Position: refs/heads/master@{#37833}
+
+PR-URL: <pr link>
+```
+* Open a PR against the `v6.x-staging` branch in the Node.js repo. Launch the normal and [V8-CI](https://ci.nodejs.org/job/node-test-commit-v8-linux/) using the Node.js CI system.
```

Comment:
v6.x-staging is just an example, right? because backports might be necessary to any LTS staging branch